Widgets/Controls/FontPreview.ascx.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace Widgets.Controls {
+    public partial class FontPreview : System.Web.UI.UserControl {
+        protected void Page_Load(object sender, EventArgs e) {
+            Button.Click += new EventHandler(this.ButtonClick);
+        }
+        void ButtonClick(object sender, EventArgs e) {
+            Paragraph.Font.Name = Fonts.SelectedValue;
+        }
```

Comment:
I think we talked about this in person, but you could remove the button from this page and move this line of code into a `Page_PreRender` method:

```
Paragraph.Font.Name = Fonts.SelectedValue;
```

Then, set your drop down list control to auto postback. Now, the font will change every time a new value is selected in the drop down list.